apps/web/frontend/components/addProjectMember.tsx

```diff
@@ -0,0 +1,74 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useForm } from 'react-hook-form'
+import { AiOutlinePlus } from 'react-icons/ai'
+import { useMutation } from 'urql'
+import { z } from 'zod'
+
+import { Button, InputField } from '@progwise/timebook-ui'
+
+import { graphql } from '../generated/gql'
+
+interface AddProjectMemberProps {
+  projectId: string
+}
+
+interface FormState {
+  email: string
+}
+
+const ProjectMembershipCreateByEmailDocument = graphql(`
+  mutation projectMembershipCreateByEmail($email: String!, $projectId: ID!) {
+    projectMembershipCreateByEmail(email: $email, projectId: $projectId) {
+      title
+      members {
+        name
+      }
+    }
+  }
+`)
+
+const formSchema = z.object({
+  email: z.string().trim().email().min(1).max(50),
+})
+
+export const AddProjectMember = (props: AddProjectMemberProps) => {
```

Comment:
```suggestion
export const AddProjectMemberForm = (props: AddProjectMemberFormProps) => {
```

...because this is a form